### PR TITLE
sort gage locations by latitude and then redistribute into separate files

### PR DESCRIPTION
**Pull request by mhines-usgs**
Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
re-add the station data but do it in order from north to south (sort by latitude desc)

Description
-----------
re-add the station data but do it in order from north to south (sort by latitude desc)

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/assets/monitoring_locations/delawareBasinCameraLocations.js [hunk 1]
```diff
@@ -106,6 +106,32 @@ export default {
                     "coordinates": [-75.0825, 40.82638889]
                 }
             },
+            {
```
Comment (mhines-usgs) [inline src/assets/monitoring_locations/delawareBasinCameraLocations.js]: heh, this activity allowed me to discover one i had missed completely 😅

File: src/assets/monitoring_locations/delawareBasinNextGenerationLocationsSorted.js [hunk 2]
```diff
@@ -0,0 +1,1098 @@
...
+                        "curve": 1,
+                        "essential": true
+                    },
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [-75.0144444, 41.56388889]
+                }
+            },
+            {
```
Comment (mhines-usgs) [inline src/assets/monitoring_locations/delawareBasinNextGenerationLocationsSorted.js:342]: @mwernimont i completely missed this location the first time somehow, it goes in your file...

Comment (mhines-usgs) [reply src/assets/monitoring_locations/delawareBasinNextGenerationLocationsSorted.js:342]: USGS-01428500

**Comment by mhines-usgs** [comment]
I used Laura's suggestion, and went back to my original Rscript where I got the metadata for the locations from, and added her sort by latitude to the function. Then I cleared out the existing features in each file (minus the enhanced js since marty already has that working ok he says) and re-added them in order.

**Review by abriggs-usgs — APPROVED** [review]
Wow, that should be a major headache saver!

**Pull request merged**